packages/backend/.eslintrc

```diff
@@ -1,3 +1,12 @@
 {
-  "extends": "../../.eslintrc"
+  "extends": "../../.eslintrc",
+  "rules": {
+    "no-restricted-syntax": [
+      "error",
+      {
+        "selector": "ImportDeclaration[specifiers.length = 0]",
+        "message": "Empty imports are not allowed"
+      }
+    ]
```

Comment:
Could this be in the repository root?

Reply:
It also affects `import <module>` such as frontend's `import styles.css` giving false-positives. For backend, it works like a charm